pkg/protocol/engine/booker/inmemorybooker/booker.go

```diff
@@ -198,30 +198,30 @@ func (b *Booker) inheritConflicts(block *blocks.Block) (conflictIDs ds.Set[iotag
 
 		switch parent.Type {
 		case iotago.StrongParentType:
-			conflictIDsToInherit.AddAll(parentBlock.ConflictIDs())
+			spendIDsToInherit.AddAll(parentBlock.SpendIDs())
 		case iotago.WeakParentType:
-			conflictIDsToInherit.AddAll(parentBlock.PayloadConflictIDs())
+			spendIDsToInherit.AddAll(parentBlock.PayloadSpendIDs())
 		case iotago.ShallowLikeParentType:
 			// Check whether the parent contains a conflicting TX,
 			// otherwise reference is invalid and the block should be marked as invalid as well.
-			if signedTransaction, hasTx := parentBlock.SignedTransaction(); !hasTx || !parentBlock.PayloadConflictIDs().Has(lo.PanicOnErr(signedTransaction.Transaction.ID())) {
+			if signedTransaction, hasTx := parentBlock.SignedTransaction(); !hasTx || !parentBlock.PayloadSpendIDs().Has(lo.PanicOnErr(signedTransaction.Transaction.ID())) {
 				return nil, ierrors.Wrapf(err, "shallow like parent %s does not contain a conflicting transaction", parent.ID.String())
 			}
 
-			conflictIDsToInherit.AddAll(parentBlock.PayloadConflictIDs())
-			//  remove all conflicting conflicts from conflictIDsToInherit
-			for _, conflictID := range parentBlock.PayloadConflictIDs().ToSlice() {
-				if conflictingConflicts, exists := b.conflictDAG.ConflictingConflicts(conflictID); exists {
-					conflictIDsToInherit.DeleteAll(b.conflictDAG.FutureCone(conflictingConflicts))
+			spendIDsToInherit.AddAll(parentBlock.PayloadSpendIDs())
+			//  remove all conflicting spends from spendIDsToInherit
+			for _, spendID := range parentBlock.PayloadSpendIDs().ToSlice() {
+				if conflictingSpends, exists := b.spendDAG.ConflictingSpends(spendID); exists {
+					spendIDsToInherit.DeleteAll(b.spendDAG.FutureCone(conflictingSpends))
 				}
 			}
 		}
 	}
 
-	// Add all conflicts from the block's payload itself.
-	// Forking on booking: we determine the block's PayloadConflictIDs by treating each TX as a conflict.
-	conflictIDsToInherit.AddAll(block.PayloadConflictIDs())
+	// Add all spends from the block's payload itself.
+	// Forking on booking: we determine the block's PayloadSpendIDs by treating each TX as a conflict.
```

Comment:
Is this still the case? I think we don't treat all TX as a conflict anymore, but as a spend.